go/simple/v2/examples/simple_example/simple_example.go

```diff
@@ -0,0 +1,113 @@
+package main
+
+import (
+	"log"
+	"os"
+
+	"github.com/joho/godotenv"
+
+	"ampli-example/ampli"
+)
+
+const userID = "ampli-go-user-id"
+
+func main() {
+	err := godotenv.Load()
+	if err != nil {
+		log.Fatalln("Error loading .env file")
+	}
+	apiKey := os.Getenv("AMPLITUDE_API_KEY")
+
+	// Initialize the Ampli instance with LoadOptions and LoadClientOptions
+	ampli.Instance.Load(ampli.LoadOptions{
+		Client: ampli.LoadClientOptions{
+			Configuration: ampli.NewClientConfig(apiKey),
+		},
+	})
+	defer ampli.Instance.Shutdown()
+
+	// Identify using IdentifyProperties in tracking plan
+	ampli.Instance.Identify(userID, ampli.Identify.Builder().
+		SetRequiredNumber(16.6).
+		SetOptionalArray([]string{"abc", "test"}).
+		Build())
+
+	// Group Identify using GroupProperties in tracking plan
+	ampli.Instance.GroupIdentify("Org", "Engineer", ampli.Group.Builder().
+		SetRequiredBoolean(true).
+		SetOptionalString("optional-string").
+		Build())
```

Comment:
Thank you @falconandy the new builders are great.

nit: can we remove the `Set` prefix similar to Java builders? E.g. 
```suggestion
	ampli.Instance.GroupIdentify("Org", "Engineer", ampli.Group.Builder().
		RequiredBoolean(true).
		OptionalString("optional-string").
		Build())
```